apps/analysis/serializers.py

```diff
@@ -92,7 +91,7 @@ def get_metadata(self, data: Any) -> ResourceMetadata:
         )
 
     def get_summary(self, data: Any) -> str:
-        if type(data) == str:
+        if isinstance(data, str):
```

Comment:
Did the linter do this?

Reply:
Oh wait, [I see it did](https://docs.astral.sh/ruff/rules/type-comparison/). Nice